src/hooks/groups/useFetchGroupsOfUser.ts

```diff
@@ -2,31 +2,20 @@ import { GROUP_URL } from '@/constants/urls'
 import { useUser } from '@/publicodes-state'
 import { useQuery } from '@tanstack/react-query'
 import axios from 'axios'
-import { useMemo } from 'react'
 
 export function useFetchGroupsOfUser() {
-  const { user, simulations } = useUser()
-
-  const groupIds = useMemo(
-    () =>
-      simulations
-        .filter((simulation) => simulation.group)
-        .map((simulation) => simulation.group),
-    [simulations]
-  )
+  const { user } = useUser()
 
   return useQuery({
-    queryKey: ['groups', user.userId, groupIds],
+    queryKey: ['groups', user.userId],
     queryFn: () =>
       axios
         .post(`${GROUP_URL}/fetch-groups`, {
           userId: user.userId,
-          groupIds,
```

Comment:
Ici tu ne passes plus du tout `groupIds` alors qu'on récupère encore  `const groupIds = req.body.groupIds` [ici](https://github.com/incubateur-ademe/nosgestesclimat-server/pull/55/files), c'est normal ?

Reply:
Yep, c'est expliqué en dessous : 
- // If no groupIds are provided, we check if the user is a participant in any group
- // If it is provided, we check if the user is a participant in the provided groups

À terme on pourra à nouveau renvoyer groupIds (vu que toutes les simulations récentes ont l'attribut group)

Reply:
Oui j'avais bien compris, mais dans ce cas étant donné que tu gères les deux cas de figure côté back pourquoi tu ne passes plus du tout `groupIds` ?